### ISSUE TITLE
Disruption after load files

### ISSUE THREAD

**Issue by ccmeth** [robinweide/GENOVA#339]
Hi, developers,

I met with an issue related to load_contacts. After I load hic matrix, the R studio became disrupted and I could do nothing, could not save the data.

![image](https://github.com/robinweide/GENOVA/assets/54224100/cd5bdf59-6d1a-4350-979a-c516bb54c27d)

Could you please give me some suggestions?
Thanks!

**Comment by teunbrand** [comment]
Hi there,
Do you have sufficient memory to load the data into the session?
Does it work for larger resolutions, like 1Mb?

**Comment by ccmeth** [comment]
> Hi there, Do you have sufficient memory to load the data into the session? Does it work for larger resolutions, like 1Mb?

1Mb also does not work for me. However, several days ago, I could used load_contacts without problem.
![image](https://github.com/robinweide/GENOVA/assets/54224100/77c57f6f-7924-477f-b502-68a9356687d7)

**Comment by ccmeth** [comment]
![image](https://github.com/robinweide/GENOVA/assets/54224100/d2484d1e-9972-4020-89c4-8497b889d284)
I don't think this issue is associated with memory size. After loading high-resolution (5kb) hic data, my free memory is still 41G,

**Comment by ccmeth** [comment]
Furthermore, previously, I also met with another open issues (cannot load the second .hic) reported by other person. The first .hic could be successfully loaded but the second failed (no error message displayed, but I could no nothing: R studio disrupted), therefore I needed to save the first matrix immediately after loading, then close R studio, restart R studio and load the second .hic files.

**Comment by teunbrand** [comment]
I'm unsure about what causes this issue.
Is it 1 file that doesn't work or other files also don't work?

> several days ago, I could used load_contacts without problem

That doesn't make it easier to debug, as it points towards something being a cause of the issue that doesn't live in genova.

**Comment by ccmeth** [comment]
Yeah, it's very strange. I tried several times but load was always not success even at low resolution (1Mb). However, another .hic files with high resolution (10kb) was success. The size of two .hic files was similar.

**Comment by ccmeth** [comment]
When the load process was not successful, I could not close R studio normally and need to quit with "force". I noticed that although R studio was closed, after checking "top" in terminal, rsession process was still ongoing.

**Comment by LucasMcNU** [comment]
I would check the number of open files limit for your RStudio Server instance. I ran into an issue where the RStudio Server I was using had a very low limit on the number of open files and then after loading the HiC file my RSession would become unusable.
